src/main.js

```diff
@@ -1,12 +1,17 @@
 // Base imports
 import Vue from 'vue'
 import vuetify from '@/plugins/vuetify'
+
 import Router from 'vue-router'
 import App from '@/App.vue'
 
 // Plugins
 import store from '@/store'
 import { defaultApolloProvider } from '@/vue-apollo'
+
+// eslint-disable-next-line
+// import okta from '@/plugins/okta'
```

Comment:
do we not need this to be active?

Reply:
Nope, this is specifically for the in-app login experience